src/main/java/org/jboss/tm/TransactionManagerLocator.java

```diff
@@ -21,166 +21,136 @@
   */
 package org.jboss.tm;
 
+import jakarta.transaction.TransactionManager;
 import java.lang.reflect.Method;
-
 import javax.naming.InitialContext;
 import javax.naming.NamingException;
-import jakarta.transaction.TransactionManager;
-
 import org.jboss.logging.Logger;
 
 /**
  * Locates the transaction manager.
- * 
- * @todo this really belongs in some integration layer with
- *       a more pluggable implementation
+ *
  * @author <a href="adrian@jboss.com">Adrian Brock</a>
- * @author <a href="mailto:galder.zamarreno@jboss.com">Galder Zamarreno</a>  
+ * @author <a href="mailto:galder.zamarreno@jboss.com">Galder Zamarreno</a>
```

Comment:
Could we get rid of these @author comment lines?

Reply:
I think you'd need to check with the original author.

Reply:
Ok, thanks.